tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md

```diff
@@ -39,17 +40,23 @@
   - [Items That DO Meet the Requirement (0)](#Items-That-DO-Meet-the-Requirement-0)
 
 
+## How to meet the requirement
 Acknowledging that a cert/key pair or CA bundle can auto-regenerate after it expires offline means
 that if the cluster is shut down until the certificate expires, when the machines are restarted
 the cluster will automatically create new cert/key pairs or update CA bundles as required without human
 intervention.
-To assert that a particular cert/key pair or CA bundle can do this, add the "certificates.openshift.io/auto-regenerate-after-offline-expiry" annotation to the secret or configmap and 
-setting the value of the annotation a github link to the PR adding the annotation.
-This assertion also means that you have
+
+To assert that a particular cert/key pair or CA bundle can do this, add the annotation to the secret or configmap.
+```yaml
+  annotations:
+    certificates.openshift.io/auto-regenerate-after-offline-expiry: https//github.com/link/to/pr/adding/annotation, "quote escaped formatted name of e2e test that ensures the PKI artifact functions properly"
```

Comment:
I think just the test name should be sufficient, we can find out which PR added it with `git blame` (also across releases)

Reply:
> I think just the test name should be sufficient, we can find out which PR added it with `git blame` (also across releases)

Based on past experience in kube with api approvals, there's nothing quite like the link embedded.